tests/sentry/monitors/endpoints/test_organization_monitor_details.py

```diff
@@ -319,6 +319,13 @@ def test_cronjob_crontab_invalid(self):
             status_code=400,
             **{"config": {"schedule": "* * * *"}},
         )
+        self.get_error_response(
+            self.organization.slug,
+            monitor.slug,
+            method="PUT",
+            status_code=400,
+            **{"config": {"schedule": "* * 31 2 *"}},
```

Comment:
is feb 31 the only example? could be useful to document some other ones if so.

Reply:
Hmm it's what we triggered an error on. I can look into other examples.